packages/react/src/index.tsx

```diff
@@ -112,26 +112,55 @@ function NFT({
   )
 }
 
+export function Embed({
+  data,
+  style,
+  className,
+  darkMode,
+  autoPlay,
+}: NFTEProps) {
+  useStyleSheet(styles)
+
+  if (!data) return <Loading style={style} />
+
+  return (
+    <NFT
+      data={data}
+      className={className}
+      style={style}
+      darkMode={darkMode}
+      autoPlay={autoPlay}
+    />
+  )
+}
+
 export function NFTE({
-  contract = "0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0",
-  tokenId = "17824",
+  contract,
+  tokenId,
   initialData,
   className,
   style,
   darkMode,
   autoPlay = true,
+  children,
 }: {
   contract: string
   tokenId: string
   initialData?: NFTData
   className?: string
   style?: CSSProperties
   darkMode?: boolean
-  autoPlay: boolean
+  autoPlay?: boolean
+  children?: (props: {
+    data: NFTData | undefined
+    className?: string
+    style?: CSSProperties
+    darkMode?: boolean
+    autoPlay?: boolean
+  }) => JSX.Element
 }) {
-  useStyleSheet(styles)
-
   const [data, setData] = useState<NFTData | undefined>(initialData)
+  const element = children || Embed
```

Comment:
I went back and forth on this, my worry is that if this logic was here then the code would still be pulled in even if the Embed component was not being used, right?

Reply:
I think that’s ok considering the size of `Embed`. Needing to import `Embed` increases the scope of use. Ideally it’s possible to initialize the component with just the `address` and `id` in the simplest configuration.

Reply:
Yeah fair point